geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/command/KeySet.java

```diff
@@ -199,4 +219,7 @@ private static void sendKeySetResponseChunk(Region region, List list, boolean la
     chunkedResponseMsg.sendChunk(servConn);
   }
 
+  public boolean isInTransaction() {
```

Comment:
Why is this public? Seems like protected or package would be enough and it is not really part of the "public" contract of a KeySet command